playwright/_impl/_page.py

```diff
@@ -199,7 +199,7 @@ def __init__(
         )
 
     def __repr__(self) -> str:
-        return f"<Page url={self.url!r}>"
+        return f"<Page url={self.url!r}> frame={self.main_frame!r}"
```

Comment:
I think the frame does not provide much value here.

Reply:
Frame has name which can be useful.